cirras-underwriting-liquibase/scripts/03_00_xx/03_05_00/01/ddl/tables/cuws.alter.declared_yield_rollup_forage.sql

```diff
@@ -0,0 +1,3 @@
+--add two more digits to the total bales columns
+
+ALTER TABLE cuws.declared_yield_field_rollup_forage ALTER COLUMN total_bales_loads TYPE numeric(6,0);
```

Comment:
I noticed that declared_yield_contract_cmdty_forage also has a total_bales_loads column. Do you think it should be expanded too?

Reply:
We should do that but in another release

Reply:
We'll expand it later, if needed.